Content/wip/hardening changes

marybyrnepark opened the pull request:
Incorporating hardening changes for Apply & Track Claims

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
burials-preneed first change

bethpotts committed:
shorten intro paragraph: PTSD

Experimenting with this to show Melissa.

marybyrnepark committed:
Update apply.md

The green button "select correct form" isn't here, I don't know why
also, "ready to apply" is on the live site but not here

marybyrnepark committed:
Update burial-costs.md

marybyrnepark committed:
Update apply-for-certificate-of-eligibility.md

marybyrnepark committed:
Update apply-vre.md

marybyrnepark committed:
Update family-members.md

marybyrnepark committed:
Update apply-vre.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update survivors-pension.md

marybyrnepark committed:
Update apply.md

Using hardening changes from PR #6981

marybyrnepark committed:
Update apply.md

Incorporating hardening changes from PR #6981

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

bethpotts committed:
Update apply.md

marybyrnepark committed:
Update apply.md

took "how do I apply" out of blue box

marybyrnepark committed:
Update apply.md

took "how do I apply" out of blue box

marybyrnepark committed:
Update apply.md

took "how do I apply" out of blue box

marybyrnepark committed:
Update apply.md

took "how do I apply" out of blue box

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
Update apply.md

bethpotts committed:
removed extra how do I apply subhead: pension

bethpotts committed:
Update ApplicationStatus.jsx

bethpotts committed:
Update apply.md

bethpotts committed:
change subhead "ready to apply" to "how do I apply?": edu

marybyrnepark committed:
Update pre-need.md

marybyrnepark committed:
changed headers to question as per our style

marybyrnepark committed:
Update EducationWizard.jsx

marybyrnepark commented:
@peggygannon could you pls proofread this PR? 
I will direct you to what you need to be looking at in the issue. Thanks!

peggygannon committed:
Proofread fix: update head

peggygannon committed:
proofread fix: update head

peggygannon committed:
proofread fix: update headers

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: reinstated dropped text

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: updated header

peggygannon committed:
proofread fix: update display title

peggygannon committed:
proofread fix: update link headers

peggygannon committed:
proofread fix: header tweak type of pensions

updated header since there are two types of pension benefits in this section

peggygannon committed:
proofread fix: typo

peggygannon committed:
proofread fix: consistency other ways to apply

marybyrnepark committed:
changed wording and subheads for consistency

marybyrnepark committed:
Update apply.md

marybyrnepark committed:
added SEO terms to eligibility link

marybyrnepark committed:
added SEO terms to eligibility link

marybyrnepark committed:
removed extra subhead

bethpotts committed:
Removed end div to try to fix left nav issue: hc apply

bethpotts committed:
Update apply.md

bethpotts committed:
Update apply.md

bethpotts committed:
Update apply.md

bethpotts committed:
Update apply.md

bethpotts committed:
Update apply.md

phiden committed:
Update apply.md

looking for a wonky tag

phiden committed:
Update apply.md

phiden committed:
Update apply.md

marybyrnepark committed:
Update index.md

bethpotts committed:
changes to page titles: top nav

marybyrnepark committed:
Update index.md

marybyrnepark committed:
Update family-members.md

marybyrnepark commented:
@peggygannon I incorporated changes into https://vetsgov-pr-7812.herokuapp.com/health-care/ (just had to change "application process" to "how to apply")
--also made some changes to simplify extraneous verbiage in https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/family-members/
Can you take a look?

peggygannon committed:
proofread tweak: fix display title

fixed display title to be consistent with other sections.

peggygannon committed:
proofread tweak: fixed link title

updated link title to be consistent with other sections.

peggygannon reviewed (commented):
@marybyrnepark Hi, Mary. Proofread for this PR is complete. I think the following pages need a further look to determine if we should tweak and/or delete one of the "how to apply" headers.

https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/apply-vre/
--We have two "how to apply..? headers. Do we want to tweak or delete one of them? 
**BETH COMMENT:** @marybyrnepark To avoid two subheads starting essentially the same way, (how to apply / how do I apply), please change "How to apply if you already have your VA disability rating" to "If you already have your VA disability rating, take these steps to apply:"

https://vetsgov-pr-7812.herokuapp.com/employment/vocational-rehab-and-employment/family-members/
--we have header: How do I get these benefits? , then below that: "how do I apply?
do we want to tweak or delete one of them?
**BETH COMMENT:** Mary, you already solved this one by removing "How do I get these benefits?" and moving up "Follow these steps to apply for Chapter 36 services:" **DONE**

https://vetsgov-pr-7812.herokuapp.com/housing-assistance/home-loans/apply-for-certificate-of-eligibility/
we have header "How do I get my COE?, **BETH COMMENT:** Mary, please change to "How do I prepare before starting my application?" Then run the next two sentences together in plain text (no bold): "Gather the information you'll need to apply for your COE. Click on the description below that matches you best to find out what you'll need. (This is almost exactly what's on the page now, but without bolding, the addition of the word "below," and the removal of the word "information")
then we have header later down the page: "How do I apply for my COE?"
Do we want to delete one of these headers?

bethpotts committed:
consistency updates to "You can also apply": HC apply

bethpotts committed:
consistency changes to "You can also apply": Disability

bethpotts committed:
add br tags: disability apply

phiden committed:
html cleanup

phiden committed:
Merge branch 'master' into content/WIP/hardening-changes

bethpotts committed:
consistency in "You can also apply": education apply

bethpotts committed:
tweak: pension apply

bethpotts committed:
consistency in "You can also apply": survivors pension apply

bethpotts committed:
Update apply.md

bethpotts committed:
tweak: housing COE

bethpotts committed:
consistency in "How do I apply?": burials pre-need

bethpotts committed:
linked "Get help filing your claim": pre-need apply

marybyrnepark committed:
removed redundant "find out if you qualify"

marybyrnepark committed:
changed "check to see" to "find out" for consistency across application pages

marybyrnepark committed:
eliminated redundant sentence

marybyrnepark committed:
took out redundant phrase

marybyrnepark committed:
made edits to "how do I prepare" for consistency

marybyrnepark committed:
shortened intro

marybyrnepark committed:
edited intro for consistency

marybyrnepark committed:
fixed typo

marybyrnepark committed:
edited / shortened intro

marybyrnepark committed:
shortened intro

peggygannon committed:
intro tweak

bethpotts committed:
Update survivors-pension.md

bethpotts committed:
add back in intro para: disability-ptsd

Waiting on changing this page since it's not an Apply / How to page

bethpotts committed:
moved link to pdf: burials-pre-need

bethpotts committed:
tweaks to "Other ways to apply:" HC apply

bethpotts committed:
Update createEducationApplicationStatus.jsx

Commit:
Define headings for save-in-progress buttons as a React prop

Commit:
Merge branch 'content/WIP/hardening-changes' of https://github.com/department-of-veterans-affairs/vets-website into content/WIP/hardening-changes

bethpotts committed:
fix related link: hc-asst-living

bethpotts committed:
fix related link lang: hc-dental-care

bethpotts committed:
fix link lang: hc-other-ins

bethpotts committed:
fix relative link: health care vision care

bethpotts committed:
fix relative link: hc blind and low vision

bethpotts committed:
fix relative link: hc-where you get care

bethpotts committed:
fix relative link: hc your care team

bethpotts committed:
updates: hc apply

bethpotts committed:
Update apply.md

bethpotts committed:
add bullets to "what if I need help?": HC eligibility

bethpotts committed:
fix related link: HC health-conditions

bethpotts committed:
Fixed related link: HC-health-conditions-service-era-cold-war

bethpotts committed:
fixed related link: HC-health-conditions-service-era-gulf-war

bethpotts committed:
Fixed related link: HC-health-conditions-service-era-iraq-war

bethpotts committed:
Fix related link: HC-health-conditions-service-era-korean-war

bethpotts committed:
Fix related link: health-conditions-operation-enduring-freedom

bethpotts committed:
Fix related link: HC-health-conditions-vietnam-war

bethpotts committed:
Fix related link: HC-health-conditions-wwII

bethpotts committed:
Fix related link: HC-health-conditions-exposure-hazardous-mat

bethpotts committed:
Fix related link: HC-health-conditions-womens-health-care-needs

bethpotts committed:
proof tweak: HC index

bethpotts committed:
Fix related link: HC-schedule-an-appt

Commit:
Update burial button heading

Commit:
Merge branch 'content/WIP/hardening-changes' of https://github.com/deptartment-of-veterans-affairs/vets-website into content/WIP/hardening-changes

bethpotts committed:
move blue box: HC apply

bethpotts committed:
update names of apply page in related links: bereavement-counseling

bethpotts committed:
update name of apply related link: disability-after-you-apply

bethpotts committed:
updated name of apply page in related link: one-year

bethpotts committed:
update apply name on related link: evidence-DRC

Commit:
Update Disability Benefits apply text

bethpotts committed:
Rewrote link lang under green apply button: dis/elig

bethpotts committed:
updated lang for how to apply: VR&E elig

bethpotts committed:
update how do I apply lang: housing-assistance/elig

bethpotts committed:
updated lang "apply in one of these ways": life-ins/options/vgli

bethpotts committed:
find lang in related link "How to apply": emp/vre/external-resources

bethpotts committed:
update lang in related apply link: emp/vre/vetsuccess

bethpotts committed:
update related link hc apply lang to "how to": emp/vre/ides

bethpotts committed:
"How to" updates to left nav: disability and pension

bethpotts committed:
"learn more about how to apply" not application process: save in progress

bethpotts committed:
"learn more about how to apply" link under green button: dis/elig

bethpotts committed:
Add "Learn more" link to apply page under green button: VRE/elig

bethpotts committed:
remove br tag after green button

bethpotts committed:
fix h3 and h4 tags for How do I apply and You can also apply

bethpotts committed:
Add "You can apply online right now.": disability/apply

bethpotts committed:
h3 to h4 for You can also apply: edu/apply

bethpotts committed:
fix hierarchy of subheads: disability/apply

bethpotts committed:
fixed subhead hierarchy: housing/home-loans/coe

bethpotts committed:
Added "You can apply online right now." before green button: VRE/apply

bethpotts committed:
add "You can apply online right now." and a space btwn links: VRE/elig

bethpotts committed:
tweak: vre elig

bethpotts committed:
Update eligibility.md

bethpotts committed:
Update eligibility.md

bethpotts committed:
Update eligibility.md

bethpotts committed:
remove br tag: vre/elig

bethpotts committed:
Change link lang to "Learn more about how to apply" under green button

This removes "application process" language. It was: "Learn more about the application process." This commit changes the eligibility pages for HC, EDU, and Pension.

bethpotts committed:
Update eligibility.md

bethpotts committed:
Changed link lang to "learn more about how to apply" under green button: edu elig

bethpotts committed:
shorten intro para: vr&e apply

bethpotts committed:
shorten intro para: family-members

bethpotts committed:
Add "You can apply online right now." above green button: vre/dep-fam

bethpotts committed:
Add green button and surrounding text to COE elig page in home loans

bethpotts committed:
fixed "you can also apply" and moved green button to primary spot: COE apply

phiden committed:
deleted src/applications/common/schemaform/save-in-progress/ApplicationStatus.jsx as it no longer exists in master to resolve merge conflict

phiden commented on content/pages/pension/index.md:
```diff
@@ -1,40 +1,208 @@
 ---
```
This is the merger version of the page. Why is it in this PR?

phiden reviewed (commented):
As soon as we figure out that merger page & the breadcrumbs, this is good to go

phiden committed:
replace pension overview iwth non-merger stuff

bethpotts committed:
fixed two teaser links to "how to apply": pension overview

bethpotts committed:
updated teaser text for survivors teaser: pension overview

Commit:
Adding linter and e2e test fixes.

bethpotts commented:
@phiden This is ready to merge.

bethpotts reviewed (approved):
This is ready to merge.
